packages/edge-gateway/src/gateway.js

```diff
@@ -274,6 +281,37 @@ async function getFromDenyList (datastore, cid) {
   )
 }
 
+/**
+ * Fetch malware state from google's malware detection API and update our DENYLIST.
+ *
+ * TODO: notify badbits
```

Comment:
We won't be able to notify badbits from here (at least in the current scenario where it requires email to be sent).  We should just rely on logs